packages/cli/src/lib/setup/setupUpload.ts

```diff
@@ -530,7 +530,8 @@ export class Upload {
 
         if (!fs.existsSync(dir)) {
             // www folder have not all adapters. So show warning only for admin folder
-            (isAdmin || (cfg && cfg.common && cfg.common.onlyWWW)) &&
+            // widgets does not have www folder, but they onlyWWW flag
+            (isAdmin || (cfg && cfg.common && cfg.common.onlyWWW && !cfg.common.visWidgets)) &&
```

Comment:
```suggestion
            (isAdmin || (cfg?.common?.onlyWWW && !cfg.common.visWidgets)) &&
```

Reply:
But I like the added comments in the original version honestly ....

Reply:
With only for admin version? yes lets keep it if its still true ;-)